source/manual/load-test.html.md

```diff
@@ -0,0 +1,116 @@
+---
+owner_slack: "#govuk-developers"
+title: Run a load test with k6
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These instructions are based on [this blog post](https://grafana.com/blog/2022/06/23/running-distributed-load-tests-on-kubernetes/).
+
+1. Log into AWS and set the context to the environment you wish to test:
+
+    ```sh
+    ENVIRONMENT=staging
+    ROLE=admin
+    AWS_REGION=eu-west-1
+
+    eval $(gds aws govuk-${ENVIRONMENT}-${ROLE} -e --art 8h)
+    kubectl config use-context $ENVIRONMENT
+    ```
+
+    > You will need to assume the `admin` role in order to deploy k6 to the cluster.
+
+1. Clone the `k6-operator` repo and make this:
+
+    ```sh
+    git clone https://github.com/grafana/k6-operator && cd k6-operator
+    make deploy
+    ```
+
+    > If you get an error similar to `invalid go version '1.22.0': must match format 1.23` then you need to upgrade `go` to the correct version.
+    > This can be done by using Homebrew, e.g. `brew install go@1.23`
+
+1. Write your test script, for example to test 3000 virtual users against the Collections app over 30 seconds (bypassing router):
```

Comment:
**suggestion:** testing scope

`against the Collections app's "/world" page`?

Reply:
Good idea.  I'll add on that clarification about the scope.